packages/ui/src/components/Modal/style.less

```diff
@@ -87,9 +90,58 @@
       padding: @condo-modal-half-padding-without-border @condo-modal-padding @condo-modal-half-padding;
       border-top: @condo-global-border-width-default solid @condo-modal-header-scrolling-border-bottom-color;
 
-      .condo-btn + .condo-btn:not(.condo-dropdown-trigger, :first-child) {
+      > .condo-btn + .condo-btn:not(.condo-dropdown-trigger) {
         margin-left: 0;
       }
     }
+
+    @media @to-tablet-large {
+      position: absolute;
+      left: 0;
```

Comment:
Can you attach a screen, how it looks like...because it's seems a bit strange right now

Reply:
screen attached in the pr description